packer/scripts/al2/al2-agent-setups.sh

```diff
@@ -31,5 +32,9 @@ sudo yum install -y gh
 
 sudo yum clean all
 
+#Set java to JDK11 so Jenkins can bootstrap it
+sudo update-alternatives --set "java"  "/usr/lib/jvm/java-11-openjdk-11.0.18.0.10-1.amzn2.0.1.x86_64/bin/java"
+sudo update-alternatives --set "javac" "/usr/lib/jvm/java-11-openjdk-11.0.18.0.10-1.amzn2.0.1.x86_64/bin/java"
+
```

Comment:
This wont work on the arm64 server.

Reply:
Right! Forgot about that. Will revert to previous state then. Moving on without jdk8 on al2 then